src/Plugin/views/field/OgMembershipBulkForm.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\og\Plugin\views\field;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\og\OgMembershipInterface;
+use Drupal\system\Plugin\views\field\BulkForm;
+
+/**
+ * Defines a user operations bulk form element.
+ *
+ * @ViewsField("og_membership_bulk_form")
+ */
+class OgMembershipBulkForm extends BulkForm {
+
+  /**
+   * {@inheritdoc}
+   *
+   * Provide a more useful title to improve the accessibility.
+   */
+  public function viewsForm(&$form, FormStateInterface $form_state) {
+    // Remove all actions related to (non-)member role.
+    foreach ($this->actions as $key => $action) {
+      if (strstr($key, '-member') !== FALSE) {
```

Comment:
This is prone to fail if one create a "Special member" role, for example. I think it is better to do `list( , , $rid) = explode('-', $key, 3) and then check that $rid is not `OgRoleInterface::ANONYMOUS` or `OgRoleInterface::AUTHENTICATED`